### DISCUSSION TITLE
HDD Error during Stage 4: terminate called after throwing an instance of 'std::filesystem::filesystem_error'

### DISCUSSION THREAD

**Discussion by francovico** [Chia-Network/chia-blockchain#6090] [Support]
Hello everybody,

Im running Chia in HiveOS Linux through CLI.
I have mounted 2 mechanical HDDs, one EXT4 as temp and one NTFS as final destination.

Everything happens normally until stage 4, where in the end I get the following error:

terminate called after throwing an instance of 'std::filesystem::filesystem_error'
  what():  filesystem error: cannot remove: Read-only file system [/media/chiatemp/plot-k32-2021-05-25-12-35-10b7bfa16c82d40441e9933b1944d39cb7e1afb4e6fcf1575d15dca46314fa54.plot.p3.t4.sort_bucket_000.tmp]
Aborted (core dumped)

Initially both disks are correctly mounted as rw. After that error, when checking the status it changes to ro.

Im attach one image of error.

I repeated the process a couple of times and I get the same error.

What could be happening?

![Chia](https://user-images.githubusercontent.com/47440458/119751100-ea36b300-be70-11eb-8301-579d2ae9c28b.png)

**Comment by BrandtH22** [comment]
@francovico , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
